index-de.html

```diff
@@ -0,0 +1,273 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>Guild Wars 2 - Gold per Hour</title>
+        <meta name="description" content="How much gold you made since you started the application">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="shortcut icon" href="favicon.ico">
+        <link rel="stylesheet" href="css/normalize.css">
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" href="css/goldPerHour.css">
+        <link rel="stylesheet" href="css/spinner.css">
+        <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,900' rel='stylesheet' type='text/css'>
+        <script src="js/vendor/fix-timer.min.js"></script>
+        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+    </head>
+    <body>
+        <!--[if lt IE 8]>
+            <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <h1>Guild Wars 2 - Gold pro Stunde</h1>
+
+        <div class="language">
+            <a href="index.html"><img src="img/us.svg" alt="English" title="English" width="20px"></a>
+            <a href="index-fr.html"><img src="img/fr.svg" alt="Français" title="Français" width="20px"></a>
+        </div>
+        </br>
+
+        <div id="intro">
+            <h2>Willkommen zu Guild Wars 2 - Gold pro Stunde</h2>
+            <p>Diese Applikation verfolgt (<em>beinahe</em>) alles, was auf deinem Account vor sich geht und zeigt an, wie viel Gold du bekommen würdest, wenn du alles verkaufen würdest, was du erhalten hast, und wie viel Gold du verloren hast durch die Items, die von deinem Account entfernt wurde (verkauft, zerstört, verbraucht, usw.).</p>
+            <p>Ich habe gesagt <em>beinahe</em>, weil ich bisher nicht Items und Gold beobachten kann, das im Handelsposten darauf wartet aufgenommen zu werden. Sobald ArenaNet eine neue API hinzugefügt, um diese Informationen zu erhalten, werde ich die Applikation updaten.</p>
+            <p>Sei dir bewusst, dass in einigen Fällen es dazu kommen kann, dass es 2 oder 3 Refreshes benötigt bis erhaltene oder verlorene Items auftauchen. Die API gibt keine Live Daten zurück, jedes mal wenn sie aufgerufen wird. Sei also bitte geduldig :)</p>
+            <hr>
```

Comment:
The text here changed since you created the PR (sorry again!), like the second paragraph has been removed. Can you check the english version and update the translation?